src/OrleansVSTools/VSProjectSiloHost/VSProjectSiloHost.vstemplate

```diff
@@ -41,9 +36,6 @@
       <package id="Microsoft.Orleans.OrleansProviders" version="1.2.0" />
       <package id="Microsoft.Orleans.OrleansRuntime" version="1.2.0" />
       <package id="Microsoft.Orleans.Server" version="1.2.0" />
-      <package id="Newtonsoft.Json" version="7.0.1" />
-      <package id="System.Collections.Immutable" version="1.1.36" />
-      <package id="System.Reflection.Metadata" version="1.0.21" />
```

Comment:
By referencing Orleans packages in the templates, they will bring along all the required dependencies? That's great :)